CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contribution Guidelines
+
+Thanks you for considering a contribution to Blended.
+
+## Getting Started
+
+Before you begin developing on Blended, you'll need to setup your environment:
+
+1. Install [NodeJS](https://nodejs.org) 6.11.1 or higher.
+2. Install [Yarn](https://yarnpkg.com)
+
+All commands should be run using yarn. If you haven't switched to [yarn](https://yarnpkg.com/en/) yet, now's a great time!
+
+> If you are familiar with npm then using yarn should be a breeze. You can keep using npm if you'd prefer but you will miss out on the safety and security of yarn
+
+Install project dependencies with:
+
+```
+yarn install
+```
+
+That's it! You're all set!
```

Comment:
@cwmanning what was your method of mirroring the blendid package for testing? this may be useful to add here

Reply:
https://yarnpkg.com/lang/en/docs/cli/link/